packages/core/src/MatchMaker.ts

```diff
@@ -65,7 +65,7 @@ export async function setup(
   /**
    * Subscribe to remote `handleCreateRoom` calls.
    */
-  subscribeIPC(presence, processId, getProcessChannel(), (_, args) => {
+  await subscribeIPC(presence, processId, getProcessChannel(), (_, args) => {
```

Comment:
what is the point of awaiting this? Is there any reason to delay the other calls?